src/views/Timesheets/index.js

```diff
@@ -595,38 +596,12 @@ const Timesheets = (props) => {
         return <GordonOffline feature="Timesheets" />;
       } else if (!isUserStudent) {
         return (
-          <Grid container justifyContent="center" spacing="16">
-            <Grid item xs={12} md={8}>
-              <Card>
-                <CardContent
-                  style={{
-                    margin: 'auto',
-                    textAlign: 'center',
-                  }}
-                >
-                  <br />
-                  <h1>{'Timesheets Unavailable'}</h1>
-                  <h4>
-                    {
-                      'Timesheets is currently available for students only. Support for staff will come soon!'
-                    }
-                  </h4>
-                  <br />
-                  <br />
-                  <Button
-                    color="primary"
-                    backgroundColor="white"
-                    variant="outlined"
-                    onClick={() => {
-                      window.location.pathname = '';
-                    }}
-                  >
-                    Back To Home
-                  </Button>
-                </CardContent>
-              </Card>
-            </Grid>
-          </Grid>
+          <GordonLimitedAvailability
+            pageName="TimeSheets"
+            backToLocation="Home"
+            backToLink=""
```

Comment:
These would make sensible default values, so that if we didn't want to redirect the user to a specific location, we could just not pass these props and the component would send them to the Home page.

Reply:
Yes, okay. I was not sure how to do that initially but I will figure it out :)

Reply:
Done!